styles.css

```diff
@@ -343,6 +391,15 @@ img.logo-img {
   padding: 0 0.5em 0 0;
 }
 
+.celebration-images {
+  display: flex;
+  float: left;
+  flex-direction: column;
+  margin-top: 10px;
+  width: 100%;
+  padding: 5em;
+}
+
 .speakers-container {
   display: flex;
   align-items: center;
```

Comment:
Great design @godybuoy 🤝

- [ ] Kindly ensure the mobile design replicates the design given in the project requirements.
- Your design
![Your-mobile-speakers](https://user-images.githubusercontent.com/63917725/235325243-ec0aec42-892b-4ff0-939e-fa81e7640852.PNG)

- Given design
![Given-mobile-speakers](https://user-images.githubusercontent.com/63917725/235325235-22026ac3-d72f-43dc-9d59-bdcf13fae2c2.PNG)